scripts/testnet/deploySovereignScreen.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
```

Comment:
I think we can rename this file to smth like "deployFullStackSovereignChainWithScreen.sh"